NOTICE

```diff
@@ -0,0 +1,284 @@
+Elastic APM Java Agent
+Copyright 2018-2019 Elasticsearch B.V.
+
+###############################################################################
+
+This product includes software developed by The Apache Software Foundation
```

Comment:
I don't think we include software developed by The Apache Software Foundation. Or do we?

Reply:
I think you are right. Not sure why I left this. Probably too many edits 🤓